README.md

```diff
@@ -9,3 +9,13 @@ To run all linters/check use:
 ```
 $ nix-build pre-commit.nix
 ```
+
+
+## Related RFCs
+Marvin is basically a trial run of rfc 30.
+- https://github.com/NixOS/rfcs/pull/30/
+
+Further readings: 
+- https://github.com/NixOS/rfcs/blob/master/rfcs/0039-unprivileged-maintainer-teams.md
+- https://github.com/NixOS/rfcs/blob/master/rfcs/0051-mark-stale-issues.md
+- https://github.com/NixOS/rfcs/blob/master/rfcs/0055-retired-committers.md
```

Comment:
This one really isn't relevant to Marvin, I'd rather leave it out.

Reply:
> Many people were given push access to the nixpkgs repository, which is kept even if these committers become inactive. This RFC proposes moving these contributors to a new team without push access.

In the sense that marvin also has a "merger" primitive (and needs a corresponding lifecycle management).
That RFC proposes a "commiter" lifecycle with push acces, so essentially it looks like marvin's "mergers" and RFC55's "commiters" might be a group ov people that overlaps.

My reasoning is: if the "commiters" can go inactive, so can "mergers"? So arguing by "commiters" necesarily beeing a superset of  "mergers" this RFC theologically might apply to marvin, as well. Don't you think?

Reply:
Final verdict? :wink:

Reply:
Yes marvin has the concept of a "merger", but marvin doesn't *grant* this permission -- it is just told about it. If a reviewer becomes inactive -- whether they have commit permission or not -- this would become problematic *much* sooner than after a year. They should be removed in that case, but since there is no permission associated with it and they can be re-added at any time there is no need for an RFC.

So in summary, I really don't see any overlap. Yes marvin needs to know if a reviewer has commit access. That might change in the future for various reasons, the "retire" RFC is one of them. But there is nothing special about it, at least concerning this bot.